src/index.js

```diff
@@ -20,7 +20,7 @@ import SelectField from './elements/SelectField/SelectField'
 import Shield from './elements/Shield/Shield'
 import Statistic from './elements/Statistic/Statistic'
 import SwitchField from './elements/SwitchField/SwitchField'
-
+import DoubleSlider from './elements/DoubleSlider/DoubleSlider'
```

Comment:
Elements should be separated from layouts by a newline and internally sorted alphabetically.